cmd/frontend/internal/cli/middleware/opengraph.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="{{.Title}}" />
+    <meta property="og:description" content="{{.Description}}" />
```

Comment:
```suggestion
    <meta property="og:description" content="{{.Description}}" />
    {{if .Lines}}
    <meta property="twitter:label1" content="Lines" />
    <meta property="twitter:data1" content="{{.Lines}}" />
    {{end}}
```

Or maybe it needs to be `{{if .Lines != ""}}`